crates/distribution-types/src/lib.rs

```diff
@@ -1077,20 +1060,18 @@ mod test {
     /// Ensure that we don't accidentally grow the `Dist` sizes.
     #[test]
     fn dist_size() {
-        // At time of writing, Unix is at 240, Windows is at 248.
         assert!(
-            std::mem::size_of::<Dist>() <= 248,
+            std::mem::size_of::<Dist>() <= 336,
```

Comment:
I have a branch boxing the entire `Dist` struct, but it didn't help with the windows debug stack overflows.